src/endpoints/http/collections/export/lib/parser.js

```diff
@@ -0,0 +1,40 @@
+import stream from 'stream';
+import jcsv from 'jcsv';
+import concat from 'concat-stream';
+import str from 'string-to-stream';
+
+class json extends stream.Transform {
+  constructor() {
+    super({ objectMode: true });
+  }
+
+  _transform(data, encoding, cb) {
+    data = JSON.stringify(data);
```

Comment:
Streams only accept strings or buffers by default and it cant be configured so we need to convert the objects into strings so that we can pipe the data